Bottom: 794a7ccd18fea22a0006c5b1e8a9c8d11e7e13b3
Top:    2a4e739b0664e4299db3dc28752b881f82f8ab62
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-22 02:56:20 +0000

Refresh of butcher-the-xor-crosshair-draw

---

diff --git a/src/outline_draw.c b/src/outline_draw.c
index e8731a0..c591146 100644
--- a/src/outline_draw.c
+++ b/src/outline_draw.c
@@ -36,6 +36,7 @@
 #include "crosshair.h"
 #include "data.h"
 #include "draw.h"
+#include "hid_draw.h"
 #include "line.h"
 #include "misc.h"
 #include "polygon.h"
@@ -257,7 +258,7 @@ outline_draw_pcb_pad (DrawAPI *dapi, LayerType *layer, PadType *pad)
   dapi->graphics->thindraw_pcb_pad (dapi->gc, pad, false, false);
 }
 
-DrawAPI *outline_draw_new (HID_DRAW_API *graphics)
+DrawAPI *outline_draw_new (HID_DRAW *graphics)
 {
   DrawAPI *dapi;
